styles.css

```diff
@@ -407,3 +427,108 @@ button[type="submit"]:hover {
 button[type="submit"]:active {
   background: #2230d2;
 }
+
+/* desktop version */
+@media only screen and (min-width: 768px) {
+  header i {
+    display: none;
+  }
+
+  nav .header-links {
+    display: flex;
+    gap: 2.5rem;
+    margin: 0 auto;
+  }
+
+  nav .header-links a {
+    font-family: "Poppins", serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    color: #344563;
+  }
+
+  .header-background {
+    background-image: url(./images/Header-bg-desktop.svg);
+  }
+
+  .about {
+    width: 70%;
+    margin-top: 7rem;
+    padding: 5rem 0 0;
+  }
+
+  .works .projects .details {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 2rem;
+  }
+
+  .works .projects .details img.pos {
+    order: 2;
+  }
+
+  .works .projects .details .card-info {
+    order: 1;
+  }
+
+  .switched-cards-info {
+    padding-top: 1rem;
+    padding-left: 1rem;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
- [ ] Nice job, your design look almost identic to Figma's design. :muscle: :sunglasses: However in order to make the "About Myself" section look exactly as the Figma design (see image), kindly make the left block takes 40% and the right block the remaining 60% of the page's width. You can rewrite this line and use `grid-template-columns: 40% 60%;`
### Figma's Design
![Screenshot from 2023-04-05 06-38-27](https://user-images.githubusercontent.com/91142432/230004083-1af654aa-b3d9-461b-ad6a-640f2c71bdee.png)

### Your Design
![Screenshot from 2023-04-05 06-56-44](https://user-images.githubusercontent.com/91142432/230005376-22c28b61-5380-40a7-a2c3-61784b47270b.png)

Reply:
Thank you for the guidance. I have implemented the changes as advised.